test/basetest/task.py

```diff
@@ -52,8 +53,9 @@ def __init__(self, taskd=None, taskw=DEFAULT_TASK):
 
         with open(self.taskrc, 'w') as rc:
             rc.write("data.location={0}\n"
+                     "hooks.location={1}\n"
```

Comment:
@tbabej Fix test suite : specify `hooks.location`, otherwise search in ~/.config/task